tasks/main.yml

```diff
@@ -52,6 +53,15 @@
     state: present
   when: ansible_distribution_release == 'stretch'
 
+- name: Install ntp client / deamon
+  apt:
+    name:
+      - "{{ base_ntp_client }}"
+    state: present
+  when:
+    - base_ntp_client is defined
+    - base_ntp_client | length > 0
```

Comment:
hm, are really both of those checks necessary? :thinking:

Reply:
I believe yes: `base_ntp_client` / `base_ntp_package` defaults to `ntp` (in `defaults/main.yml`). But in case one really do *not* want to install an ntp package, one has to set an empty string. That's why I also check for a non-empty string too...